### PR TITLE
Add New Keyboard Exnumpad

### PR DESCRIPTION
**Pull request by Haf0**
## Description

Add exnumpad

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/androdi/exnumpad/rules.mk [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+#
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no        # Console for debug
+COMMAND_ENABLE = no        # Commands for debug and configuration
+NKRO_ENABLE = no            # Enable N-Key Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+AUDIO_ENABLE = no           # Audio output
```
Comment (zvecr) [inline keyboards/androdi/exnumpad/rules.mk:12]:
```suggestion
# This file intentionally left blank
```

File: keyboards/androdi/exnumpad/keymaps/via/keymap.c [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+     * ├───┼───┼───┼───┤
+     * │ 7 │ 8 │ 9 │   │
+     * ├───┼───┼───┤ + │
+     * │ 4 │ 5 │ 6 │   │
+     * ├───┼───┼───┼───┤
+     * │ 1 │ 2 │ 3 │   │
+     * ├───┴───┼───┤Ent│
+     * │ 0     │ . │   │
+     * └───────┴───┴───┘
+     */
```
Comment (zvecr) [inline keyboards/androdi/exnumpad/keymaps/via/keymap.c:31]: Doesnt match
```suggestion
```

File: keyboards/androdi/exnumpad/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+     * │ 7 │ 8 │ 9 │   │
+     * ├───┼───┼───┤ + │
+     * │ 4 │ 5 │ 6 │   │
+     * ├───┼───┼───┼───┤
+     * │ 1 │ 2 │ 3 │   │
+     * ├───┴───┼───┤Ent│
+     * │ 0     │ . │   │
+     * └───────┴───┴───┘
+     */
+    [0] = LAYOUT_normal_numpad(
```
Comment (zvecr) [inline keyboards/androdi/exnumpad/keymaps/default/keymap.c:33]:
```suggestion
    [0] = LAYOUT_numpad_5x4(
```

File: keyboards/androdi/exnumpad/info.json [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+    },
+    "matrix_pins": {
+        "cols": ["C0", "C1", "C2", "C3"],
+        "rows": ["B1", "B0", "D7", "D6", "D4"]
+    },
+    "diode_direction": "COL2ROW",
+    "processor": "atmega328p",
+    "bootloader": "usbasploader",
+    "layouts": {
+        "LAYOUT_normal_numpad": {
```
Comment (zvecr) [inline keyboards/androdi/exnumpad/info.json:19]:
```suggestion
        "LAYOUT_numpad_5x4": {
```

File: keyboards/androdi/exnumpad/info.json [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+        "max_power": 100
+    },
+    "matrix_pins": {
+        "cols": ["C0", "C1", "C2", "C3"],
+        "rows": ["B1", "B0", "D7", "D6", "D4"]
+    },
+    "diode_direction": "COL2ROW",
+    "processor": "atmega328p",
+    "bootloader": "usbasploader",
+    "layouts": {
```
Comment (zvecr) [inline keyboards/androdi/exnumpad/info.json:18]:
```suggestion
    "community_layouts": ["ortho_5x4", "numpad_5x4"],
    "layouts": {
```

File: keyboards/androdi/exnumpad/README.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
+# EXNUMPAD VER. 2
```
Comment (zvecr) [inline keyboards/androdi/exnumpad/README.md:1]: Should follow <https://github.com/qmk/qmk_firmware/blob/master/data/templates/keyboard/readme.md>, and file should be lowercase.

**Review by zvecr — CHANGES_REQUESTED** [review]

**Review by dunk2k — CHANGES_REQUESTED** [review]

**Pull request closed without merging**